Tests/XcodeprojTests/FunctionalTests.swift

```diff
@@ -147,8 +147,10 @@ func XCTAssertXcodeBuild(project: AbsolutePath, file: StaticString = #file, line
         if env["TOOLCHAINS"] == nil {
             env["TOOLCHAINS"] = "default"
         }
+        let schemeName = String(project.basename.dropLast(10)) + "-Package"
+        let buildDir = project.parentDirectory.appending(component: "build").asString
         try Process.checkNonZeroExit(
-            args: "xcodebuild", "-project", project.asString, "-alltargets", environment: env)
+            args: "xcodebuild", "-project", project.asString, "-scheme", schemeName, "-derivedDataPath", buildDir, environment: env)
```

Comment:
The target doesn't build, it fails with linking errors. Only autocompletion works. 

I wouldn't want to put dependency manifests because we don't want people editing them without the edit mode.